### PR TITLE
Add Builds & Terraform Update 

### PR DESCRIPTION
**Pull request by vyasworks**

### CODE REVIEW

**Review by sunghwan-park-fw — APPROVED** [review]

File: .github/workflows/pr_check_tf.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        with:
+          terraform_version: ${{ env.TF_VERSION }}
+          cli_config_credentials_token: ${{ secrets.TFC_TEAM_TOKEN }}
+
+      - name: Terraform Plan
+        id: plan
+        run: make plan
+        continue-on-error: true
+
+      # From:
```
Comment (hitankar-freshworks) [inline .github/workflows/pr_check_tf.yml:59]: something we dont need, we can remove

File: .github/workflows/pr_checks_web.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      - name: Run formatting check
+        run: yarn format:check
+
+      - name: Lint web package
+        run: yarn workspace @symchk/web lint
+
+      - name: Build web package
+        run: yarn workspace @symchk/web build
+
+      # TODO: Unit Test
```
Comment (hitankar-freshworks) [inline .github/workflows/pr_checks_web.yml:33]: Not a fan of commented code in main

**Review by hitankar-freshworks — APPROVED** [review]
LGTM, few comments

File: .github/workflows/pr_check_tf.yml (hunk 1 above)
Comment (vyasworks) [reply .github/workflows/pr_check_tf.yml:59]: which part is not needed?

**Pull request merged**